socorro/external/es/index_cleaner.py

```diff
@@ -37,11 +37,17 @@ def __init__(self, config):
         super(IndexCleaner, self).__init__()
         self.config = config
 
-    def delete_old_indices(self):
-        now = utc_now()
-        policy_delay = datetime.timedelta(weeks=self.config.retention_policy)
-        time_limit = (now - policy_delay).replace(tzinfo=None)
+    def delete_indices(self, predicate=None):
+        """Delete crash indices that match the given predicate.
 
+        :param predicate:
+            A callable of the form ``predicate(index)``, where ``index``
+            is a string containing the name of the index. If the
+            callable returns true, the index will be deleted.
+
+            The default is None, which deletes all crash indices.
```

Comment:
We've been using `:arg ARG: some docs here` format and recently, I've been doing `:arg TYPE ARG: some docs here` in cases where the type information is helpful.

* https://github.com/mozilla-services/socorro/blob/master/socorro/signature/generator.py#L85
* https://github.com/mozilla-services/socorro/blob/master/socorro/lib/treelib.py#L163

It's not a big deal, but if you're a stickler for style uniformity, then that's what we've been doing.